index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <head>
+        <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.2/css/all.css" integrity="sha384-*********************" crossorigin="anonymous">
+      </head>
+      
+    <title>Portfolio</title>
+</head>
+<body>
+   <div class="top-navigation">
+    <nav class="nav-bar">
+        <ul>
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+        </ul>
+    </nav>
+   </div>
+
+   <div class="about-me">
+    <h1> I'm Massa Zoumanigui <br> glad to see you</h1>
+    <p>I’m a software developer! I can help you build a product , feature or website Look through <br> 
+        some of my work and experience! If you like what you see and have a project you need <br>
+        coded, don’t hestiate to contact me</p>
+        <h2 class="CONNECT"><a href="#">LET'S CONNECT</a> </h2>
+        <button class="my-button">Click me</button>
+        <ul>
+            <li> <a href="#" target="_blank"><i class="fab fa-twitter"></i></a></li>
+            <li><a href="#" target="_blank"><i class="fab fa-git hub"></i></a></li>
+            <li><a href="#" target="_blank"><i class="fab fa-facebook"></i></a></li>
+        </ul>
+      
+
+
+       
+
+        
+   </div>
```

Comment:
Nice work setting up the intro section👍, I think we can make this better by wrapping our content section in a `main` tag😉 This will make our project more semantic🤗
ALso, your intro section looks a bit different from the given design and we are required in this milestone to stick to the given deisgn😉

Your intro
<img width="311" alt="your_full intro" src="https://user-images.githubusercontent.com/64643995/219603382-1cd0bcae-ab83-47dc-a427-b839bb4255c7.png">

design
<img width="260" alt="design_full page" src="https://user-images.githubusercontent.com/64643995/219603435-1817c4b1-c1ba-4bbe-9952-363b5e1147cb.png">

also take note of your social links and your button as it looks different from the given design😉
<img width="358" alt="your_introsectiontext" src="https://user-images.githubusercontent.com/64643995/219603578-e57dae80-4d72-4a76-9485-dc99d68309dd.png">


As you make your changes😉 take note of the font-properties, the colors used, the spacings and the background image, and ensure your project looks like the design🤗🤗🤗🤗